### DISCUSSION TITLE
Photoview Version not visible...

### DISCUSSION THREAD

**Discussion by madcoda9000** [photoview/photoview#520] [Q&A and Troubleshooting] [answered]
Hello,

could it be my fault, that the app version and build date is not shown..??

Greetings 
Sascha
![2021-09-14 11_55_03-Window](https://user-images.githubusercontent.com/25504739/133237337-db2900dc-d154-410f-9bbc-8cc2cadd1560.png)

**Comment by viktorstrate** [comment, marked as answer]
If you build it manually the version is not shown, that only works with Docker.

**Reply by madcoda9000** [reply]
ok, thank you.
